artifactory/commands/transferfiles/utils.go

```diff
@@ -300,9 +302,22 @@ func shouldStopPolling(doneChan chan bool) bool {
 	return false
 }
 
+func uploadChunkWhenPossibleHandler(phaseBase *phaseBase, chunk UploadChunk, uploadTokensChan chan string, errorsChannelMng *ErrorsChannelMng) parallel.TaskFunc {
+	return func(threadId int) error {
+		logMsgPrefix := clientUtils.GetLogMsgPrefix(threadId, false)
+		log.Debug(logMsgPrefix + "Handling chunk upload")
+		shouldStop := uploadChunkWhenPossible(phaseBase, chunk, uploadTokensChan, errorsChannelMng)
+		var err error
+		if shouldStop {
+			err = errors.New("transfer has been stopped due to user cancellation or an error has occurred")
```

Comment:
The above message says that the transfer may also have been stopped due to an error, and not necessarily by a user who stopped it. Are you sure this can happen in this case? I'm asking beucase we're returning error only if `shouldStop` is true.
Yuo can perhaps verify this by temporarily manipulating the code, and intentionally returning an error from deep within the stack trace, and see if `shouldStop` is indeed true in that case.

Reply:
This error caused a repeating message that we saw in the logs. ShouldStop is triggered if one of the following happens:
// * Error occurred while handling errors (for example - not enough space in file system)
// * Error occurred during delayed artifacts handling
// * User interrupted the process (ctrl+c)

The specific error message is inserted into the error channel